src/request-pipeline/utils.ts

```diff
@@ -17,6 +17,13 @@ import { PassThrough } from 'stream';
 import { getText, MESSAGE } from '../messages';
 import logger from '../utils/logger';
 
+const NODE_MAX_HEADER_SIZE = 81920;
+
+function recommendMaxHeadersSize (currentHeaderSize: number, headerSizeMultiplier: number = 2, headerSizePrecision: number = 2): number {
```

Comment:
```js
/* Calculates the HTTP header size that customer should specify via 
--max-http-header-size option so that the proxy can process its site
 https://nodejs.org/api/cli.html#cli_max_http_header_size_size
*/
function getRecommendedHeaderSize(currentHeaderSize: number): number {}
```